packages/hooks/src/useIsGrowthbookLoaded.ts

```diff
@@ -5,16 +5,22 @@ import { useRemoteConfig } from '@deriv/api';
 const useIsGrowthbookIsLoaded = () => {
     const [isGBLoaded, setIsGBLoaded] = useState(false);
     const { data } = useRemoteConfig(true);
+    const [isGBNotAvailable, setIsGBNotAvailable] = useState<boolean>(false);
 
     useEffect(() => {
         let analytics_interval: NodeJS.Timeout;
+
+        if (/^(localhost|127\.0\.0\.1|::1)(:\d+)?$/i.test(window.location.hostname)) {
```

Comment:
I'm wondering if indeed thats okay - it pretty much means that we can no longer test GB locally whatsover, also, if there are any paths of code reachable only when GB is enabled, they will never be called on localhost,

Reply:
I will remove this code, then for every developers should have .env with GB development variables in their local